### ISSUE TITLE
Messages in channel take a few seconds to display, on startup

### ISSUE THREAD

**Issue by holmesworcester** [ZbayApp/monorepo#383]
Steps to reproduce:

1. join a community with lots of messages in general
2. sync messages
3. restart quiet

expected: when the initial loading spinner stops, I can see a channel with messages in it

actual: loading spinner stops, I see channel, but no messages. there is no loading icon and messages take a few seconds to display. this makes the app feel like it might be broken.

it's possible that #386 will fix this do we should do that first.

We should write an e2e test for this to make sure messages appear immediately once channel view displays, assuming they've already been replicated.

**Comment by holmesworcester** [comment]
We decided that if it's helpful for determining when to hide a "loading" spinner we can guarantee that all channels have at least one message by sending a message to the channel on creation. 

The message does not need to be hidden or special in any way. It can be "Created #channel-name".

Then we can hide the loading spinner once we have loaded at least one message.

A user sending a message to the channel should also make the loading spinner go away.
